README.md

```diff
@@ -34,10 +33,7 @@ To retrieve the freshly generated DID:
 
 ### Requirements
 
-* Linux
-  * g++ to build native binaries
-* Mac OS X
-  * X Code 9 to build native binaries
+* Linux or Mac OS X
```

Comment:
If compilers are no longer needed, perhaps there is no platform restriction anymore beyond Node.js.